src/components/notFound/index.js

```diff
@@ -1,17 +1,25 @@
+import { Link } from 'react-router-dom';
 import React from 'react';
 import { translate } from 'react-i18next';
-import Box from '../box';
 import styles from './notFound.css';
-import cubeImage from '../../assets/images/dark-blue-cube.svg';
+import { PrimaryButtonV2 } from '../toolbox/buttons/button';
+import Illustration from '../toolbox/illustration';
+import routes from '../../constants/routes';
+
+const NotFound = ({ t }) => (
+  <section className={styles.notFoundPage}>
+    <div className={styles.errorMessageContainer}>
+      <Illustration name={ 'pageNotFound' }/>
+      <h2 className='empty-message'>{t('Whoops, that page is gone.')}</h2>
+      <p>{t('Sorry, we couldn’t find the page you were looking for. ' +
+        'We suggest that you return to the main dashboard.')}</p>
```

Comment:
Splitting string in `t(...)` results in it not being recognized and put into `i18n/locales/en/common.json`

Reply:
done